apps/condo/domains/acquiring/access/ExportPaymentsService.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Generated by `createservice billing.ExportPaymentsService`
+ */
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+const get = require('lodash/get')
+const { checkOrganizationPermission } = require('@condo/domains/organization/utils/accessSchema')
+const { find } = require('@core/keystone/schema')
+const { checkRelatedOrganizationPermission } = require('../../organization/utils/accessSchema')
+
+async function canExportPaymentsToExcel ({ args: { data: { where } }, authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin) return true
+
+    const permissionToCheck = 'canReadPayments'
+
+    const organizationId = get(where, ['organization', 'id'])
+    if (organizationId) {
+        return await checkOrganizationPermission(user.id, organizationId, permissionToCheck)
+    }
+
+    const organizationWhere = get(where, 'organization')
+    if (!organizationWhere) {
+        return false
+    }
+
+    const [organization] = await find('Organization', organizationWhere)
```

Comment:
I guess `deletedAt: null` should be overwritten here...so you should specify it explicitly...

Reply:
fixed